src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -1414,7 +1423,7 @@ private IEnumerable<ITypeSymbol> InferTypeInMemberAccessExpression(MemberAccessE
                     return InferTypes(awaitExpression.Expression);
                 }
 
-                return SpecializedCollections.EmptyEnumerable<ITypeSymbol>();
+                return InferTypes(expression);
```

Comment:
if that's the case, i would say that we should only do this if the previous token (which should be passed in) is the dot of the member access expression.  i.e.  if we have:

```something = MemberAccess.$$```

Then we can infer an appropriate type for the $$ position.